tests/fixtures/requests/request-test.yml

```diff
@@ -0,0 +1,30 @@
+-
+  input:
+    request:
+      linguisticRequest:
+        -
+          contacts:
+            contact:
+              -
+                UserId: '1'
+                RoleCode: AUTHOR
+              -
+                UserId: '2'
+                RoleCode: EDITOR
+          generalInfo:
```

Comment:
I'm a bit concerned about having different capitalization here, I mean:  `contacts` vs `UserId`, like this array will basically be the way external software will construct the request. Would it be too much of a mess to make this all consistent? Like I'd go for `Contacts`, for example, so camel case everywhere.

Reply:
Fixed by adding a Camel Case To Snake converter in the serializer.